packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4116,7 +4119,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
 
   /// A short, textual description of this element.
   @override
-  String toStringShort() => widget.toStringShort();
+  String toStringShort() => _widget?.toStringShort() ?? 'Defunct Element';
```

Comment:
nit: do you want to use ${this.runtimeType} to make this toString a little more specific?

Reply:
runtimeType is slow. We have a helper method that uses it in debug mode but not in profile/release mode.

I believe `toStringShort` will flow through that helper method in most cases, unless someone specifically avoids it.

Reply:
Changed this to make it flow through the helper.